common/cuda_hip/log/batch_logger.hpp.inc

```diff
@@ -2,19 +2,6 @@
 //
 // SPDX-License-Identifier: BSD-3-Clause
 
-#ifndef GKO_COMMON_CUDA_HIP_LOG_BATCH_LOGGER_HPP_INC_
```

Comment:
I do not understand this. It makes hpp to hpp.inc.
I think the original should already be unified.

Reply:
do you give the wrong direction of changes?
I have seen the batch_logger.hpp.inc without the namespace in the develop

Reply:
some changes were reverted because `batch` was too messy to unify directly without a lot of additional changes

Reply:
it makes review a bit hard. Some are based on the changes which are not reviewed yet.

Reply:
You can review it in two parts by excluding the reverting commit https://github.com/ginkgo-project/ginkgo/pull/1516/commits/f3c1a103951d54291d66e96aa1d2049b44b5f01f:

* https://github.com/ginkgo-project/ginkgo/pull/1516/files/1e5cdf70fa140f25d23236a38dbff833cd9c8f81..a52527011f8c6f80bbf581aad6d83ee5d004b7bc
* https://github.com/ginkgo-project/ginkgo/pull/1516/files/f3c1a103951d54291d66e96aa1d2049b44b5f01f..15ea33e393e6c014b772f5ee47c13f609a8146db